examples/Makefile

```diff
@@ -150,14 +150,23 @@ get-php:
 	kubeless function deploy get-php --trigger-http --runtime php7.1 --handler helloget.foo --from-file php/helloget.php
 	echo "curl localhost:8080/api/v1/proxy/namespaces/default/services/get-php/"
 
+get-php-verify:
+	kubeless function call get-php | egrep "hello world"
+
 get-php-deps:
 	kubeless function deploy get-php-deps --trigger-http --runtime php7.1 --handler hellowithdeps.foo --from-file php/hellowithdeps.php --dependencies php/composer.json
 	echo "curl localhost:8080/api/v1/proxy/namespaces/default/services/get-php-deps/"
 
+get-php-deps-verify:
+	kubeless function call get-php-deps | egrep "hello world"
```

Comment:
instead of printing hello world for the test you can check the logs of the pod:
```
kubectl logs -l function=get-php-deps | egrep ...
```

Reply:
You are totally right, let me change it.